.github/workflows/ci.yml

```
@@ -152,15 +152,15 @@ jobs:
             os: ubuntu
             experimental: false
           - os: macos
-            pyver: "3.11.0-alpha - 3.11.0"
+            pyver: "3.12-dev"
```

The CI is not even starting any tests. Maybe this is not a valid pyver value? Probably better to use a similar range to what we had before.

---

This has recently changes in the setup-python action, we can now use the normal version indicator.

```suggestion
            pyver: "3.12"
```